sdk/metric/aggregator/minmaxsumcount/mmsc.go

```diff
@@ -16,26 +16,25 @@ package minmaxsumcount // import "go.opentelemetry.io/otel/sdk/metric/aggregator
 
 import (
 	"context"
+	"sync"
 
 	"go.opentelemetry.io/otel/api/core"
 	"go.opentelemetry.io/otel/api/metric"
 	export "go.opentelemetry.io/otel/sdk/export/metric"
 	"go.opentelemetry.io/otel/sdk/export/metric/aggregator"
-	"go.opentelemetry.io/otel/sdk/internal"
 )
 
 type (
-	// Aggregator aggregates measure events, keeping only the max,
+	// Aggregator aggregates measure events, keeping only the min, max,
 	// sum, and count.
 	Aggregator struct {
-		// states has to be aligned for 64-bit atomic operations.
-		states [2]state
-		lock   internal.StateLocker
-		kind   core.NumberKind
+		lock       sync.Mutex
+		current    state
+		checkpoint state
+		kind       core.NumberKind
 	}
 
 	state struct {
-		// all fields have to be aligned for 64-bit atomic operations.
 		count core.Number
```

Comment:
It seems like this could get changed to a `uint64` in a future PR.

Reply:
Agreed.  But keeping it as core.Number here since it was that way prior to the unstatelocking.